### ISSUE TITLE
Create a landing page infrastructure

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#87]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

To have an infrastructure to create basic landing pages, without the use of page templates.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by fvolner** [comment]
Try and use the MS localization API for independent text strings, just as we recommend in our [documentation](https://docs.xperience.io/developing-websites/developing-xperience-applications-using-asp-net-core/localizing-asp-net-core-projects#LocalizingASP.NETCoreprojects-Localizingsitecontent).

Use [tag helpers](https://docs.xperience.io/developing-websites/developing-xperience-applications-using-asp-net-core/reference-xperience-tag-helpers) in views.

**Comment by JanLenoch** [comment]
Resolved via c083c56294844aec559ea62d1e440127afa590d5
